assets/styles/elements/_input.css

```
@@ -31,18 +31,20 @@ label:after {
-  border: 2px solid var(--text-color-secondary);
+  border: 2px solid var(--ui-text-secondary);
 }
 
 /* Checkbox */
 input[type="checkbox"] + label:after {
   content: "\2714";
-  color: var(--brand-secondary);
+  color: var(--ui-text-secondary);
   line-height: 1.5;
   text-align: center;
+  @media (--dark) {
```

Biome (2.1.2)

[error] Don't use unknown media feature names.

Unexpected unknown media feature name.
You should use media feature names defined in the CSS Specifications.

(lint/correctness/noUnknownMediaFeatureName)